adc/producer.py

```diff
@@ -23,14 +23,15 @@ def __init__(self, conf: 'ProducerConfig') -> None:
         self._producer = confluent_kafka.Producer(conf._to_confluent_kafka())
 
     def write(self,
-              msg: Union[bytes, 'Serializable']) -> None:
+              msg: Union[bytes, 'Serializable'],
+              headers: Optional[Union[dict,list]]=None) -> None:
```

Comment:
```suggestion
              headers: Optional[Union[dict,list]] = None) -> None:
```

Small nitpick, this just adds the spacing as is seen in other type annotations in this codebase.

Reply:
Oops, sorry, fixed now.